.github/workflows/tests.yml

```diff
@@ -20,10 +20,10 @@ jobs:
         uses: actions/cache@v2
         with:
           path: Carthage
-          key: ${{ runner.os }}-carthage-${{ hashFiles('**/Cartfile.resolved') }}
+          key: ${{ runner.os }}-carthage-${{ hashFiles('**/Cartfile.resolved') }}-3
```

Comment:
Apparently there's [no way to clear the cache](https://github.com/actions/cache/issues/2), so I just updated the key. The cache had regular frameworks instead of xcframeworks.